rector.php

```diff
@@ -102,18 +104,32 @@
         'rex_form_prio_element' => Form\Field\PriorityField::class,
         'rex_form_perm_select_element' => Form\Field\PermissionSelectField::class,
         'rex_form_raw_element' => Form\Field\RawField::class,
+        'rex_formatter' => Util\Formatter::class,
         'rex_i18n' => Translation\I18n::class,
+        'rex_log_entry' => Util\LogEntry::class,
+        'rex_log_file' => Util\LogFile::class,
+        'rex_logger' => Util\Logger::class,
```

Comment:
Ich glaube, ich würde die drei eher in einen eigenen Namespace packen. Vorschlag `Redaxo\Core\Log\`